### DISCUSSION TITLE
Hundreds of heic files missing hidden, missing thumbnails.

### DISCUSSION THREAD

**Discussion by cvinhaes** [photoprism/photoprism#2785] [Q&A] [answered]
After using photoprism successfully for a while, now I have over 800 heic (I've checked many and they all seem to be heic files from the iPhone) which ended up in the hidden folder after indexing and show a glyph placeholder instead of the pic / video thumb. I realize at some point that I had the flag to ignore heic conversion and disable that option. After that I managed to import and convert other hundreds of heic files but the hidden ones remained. I run CLI options to generate thumbnails, purge, cleanup and index but nothing fixed the problem. Commands to create thumbs and purge raise a lot of messages like the ones below:
INFO[2022-10-17T00:26:03Z] index: found no jpeg for photo uid princxb3e5rb15k4 
INFO[2022-10-17T00:26:03Z] index: found no jpeg for photo uid princxb27mm2oel4 
INFO[2022-10-17T00:26:03Z] index: found no jpeg for photo uid princxb1s5dgcas0 
INFO[2022-10-17T00:26:03Z] index: found no jpeg for photo uid princxb2bzcufsns 

I've checked some of these, which seem to match the hidden pictures. The connected heic files do exist, look good when opened and are in the proper location. Just to be on the safe side while troubleshoooting, I've checked permissions in all folders touched by photoprism and made them all 777. I'm running dockerized PhotoPrism® CE Build 220901-f493607b0, with the standard image and the docker-compose file provided mostly untouched.

Other than the missing photos reported, I could not find any other relevant error in the logs. Ideas?

**Comment by lastzero** [comment, marked as answer]
The HEIC converter had problems which will be fixed with the next update. See issue comments for details:
- https://github.com/photoprism/photoprism/issues/2726

Since we are working on multi-user support at the same time, the update takes a little longer than usual.
